expectations-of-foundation.md

```diff
@@ -0,0 +1,20 @@
+# Expectations
+With the intent to merge the Node.js Foundation and JS Foundation being announced, those interested in the continued support of Node.js and JS-related projects are working through expectations and structure for what the potential new foundation could look like. 
+
+This document is for the initial phase of work for the Bootstrap team to draft requirements that overlap with scope of the Board in order to provide an outline and structure for the foundation attuned to the needs of the projects and ecosystem stakeholders who have participated.
+
+- Communicated scope for the foundation and board
+- Annual reviews of staff, NPS, board, programs, support services to projects for accountability and incremental improvement
+- The foundation staff, executive director, and board should follow a "servant leadership" style.
+- Project representation on the Board
+  - Projects could have tiers, only 'platinum' projecs should sit on the Board directly. Other projects would have shared seats(like gold/silver member strcuture)
```

Comment:
Projects could have tiers, only 'platinum' **projects** should sit on the Board directly. Other projects would have shared seats(like gold/silver member **structure**)

Reply:
I think we are better off keeping this framed as community representation rather than project. But we do need to reach consensus on this